package.json

```diff
@@ -54,7 +54,7 @@
     "jsonc-parser": "^2.0.2",
     "lint-staged": "^8.1.0",
     "monaco-editor": "^0.16.2",
-    "monaco-editor-core": "0.15.5",
+    "monaco-editor-core": "0.16.1",
```

Comment:
I've only made a minimum viable change (MVC) to upgrade `monaco-editor-core` to 0.16.1. I will also try out (in a separate PR) if it works fine with the latest version 0.20.0 and then upgrade monaco dependencies here separately.